graphx/src/main/scala/org/apache/spark/graphx/VertexRDD.scala

```diff
@@ -128,7 +128,7 @@ abstract class VertexRDD[VD](
    *
    * @param other the other RDD[(VertexId, VD)] with which to diff against.
    */
-  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD]
+  def diff(other: RDD[(VertexId, VD)]): VertexRDD[VD] = ???
```

Comment:
Ah BTW I'm curious what this changes, and I wonder why it would resolve a MiMa warning? You're on top of it. I just want to make sure we're not missing anything.